operator/config/samples/core_v1alpha1_transactioninvoke_zeto_register_anon_enc.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   node: node1
   from: zeto.operator
   txType: public
-  toTemplate: '{{`{{ index .status.resolvedContractAddresses "zeto-factory" }}`}}'
+  toTemplate: '{{ index .status.resolvedContractAddresses "zeto-factory" }}'
```

Comment:
This is great to me - bc its not concerned with Helm support it can be a bit more readable to users.